pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -248,11 +248,11 @@ func (m *manager) GetMemoryNUMANodes(pod *v1.Pod, container *v1.Container) sets.
 	}
 
 	if numaNodes.Len() == 0 {
-		klog.V(5).InfoS("No allocation is available", "pod", klog.KObj(pod), "containerName", container.Name)
+		klog.V(5).InfoS("No allocation is available", "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name)
```

Comment:
While I'm in favor of this change, I can't help wondering if logging _both_ pod namespace/name and uid is not too much.
I think we need both to correctly correlate, but adding always both feels heavy-handed.
When touching logs, adding is almost always easier than removing, so in general I'd be prudent to add stuff, even stuff which is likely to be useful.

I'm still making  my mind as I face similar dilemmas in my https://github.com/kubernetes/kubernetes/pull/125912

Reply:
Sure, I understand. My thought process was that one of the key features of structural logging is to allow users to filter and because logs are sprinkled with both (albeit not always in the same log) the user might choose to use one over the other and we should allow that. Having said that, I am open to suggestions in this area. One thing I want to ensure is consistency across the managers so let's perhaps agree to _a_ way and stick to that in all the managers.